src/test/Claiming/Claiming.t.sol

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "forge-std/Vm.sol";
+import "@contracts/MetricToken.sol";
+import "@contracts/Protocol/QuestionAPI.sol";
+import "@contracts/Protocol/ClaimController.sol";
+import "@contracts/Protocol/QuestionStateController.sol";
+import "@contracts/Protocol/BountyQuestion.sol";
+import "@contracts/Protocol/ActionCostController.sol";
+import "@contracts/Protocol/Vault.sol";
+import {NFT} from "@contracts/Protocol/Extra/MockAuthNFT.sol";
+
+import "../../contracts/Protocol/Enums/ActionEnum.sol";
+import "../../contracts/Protocol/Enums/VaultEnum.sol";
+import "../../contracts/Protocol/Enums/QuestionStateEnum.sol";
+import "../../contracts/Protocol/Enums/ClaimEnum.sol";
+
+contract ClaimTest is Test {
+    // Roles
+    bytes32 public constant PROGRAM_MANAGER_ROLE = keccak256("PROGRAM_MANAGER_ROLE");
+    bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
+
+    // Accounts
+    address owner = address(0x0a);
+    address other = address(0x0b);
+    address other2 = address(0x0e);
+    address manager = address(0x0c);
+    address treasury = address(0x0d);
+    address other3 = address(0x0f);
+
+    MetricToken _metricToken;
+    QuestionAPI _questionAPI;
+    BountyQuestion _bountyQuestion;
+    ClaimController _claimController;
+    ActionCostController _costController;
+    QuestionStateController _questionStateController;
+    Vault _vault;
+    NFT _mockAuthNFTManager;
+    NFT _mockAuthNFTAdmin;
+
+    /// @notice Emitted when a question is created.
+    event QuestionCreated(uint256 indexed questionId, address indexed creator);
```

Comment:
can these be imported instead of recreated?

Reply:
Yep, will do that together with the rest!